docs/langref/relay/expressions.rst

```diff
@@ -0,0 +1,162 @@
+===========
+Expressions
+===========
+
+The Relay IR is a pure, expression oriented language, with a dataflow fragment
+and structured control flow. Although Relay's representation is a tree, it is
+possible to view the dataflow fragments as graph for purposes of writing and
+expressing transformations.
+
+The below sections make an attempt to clearly split the dataflow
+fragment from the control fragment.
+
+====================
+Dataflow Expressions
+====================
+
+First we will cover the set of nodes which do not involve control flow,
```

Comment:
the comma after "control flow" should be a semicolon (or the clauses should be made separate sentences entirely)